amundsen_application/static/js/ducks/search/reducer.ts

```diff
@@ -61,27 +63,34 @@ export default function reducer(state: SearchReducerState = initialState, action
   switch (action.type) {
     // Updates search term to reflect action
     case SearchAll.ACTION:
+    case SearchResource.ACTION:
```

Comment:
Because searching a resource doesn't change the search_term, there's no need to update it here. We can keep out logic cleaner and more intentional by instead having:
```
case SearchResource.ACTION:
    return {
        ...state,
        isLoading: true,
     }
```